src/packages/@ncigdc/modern_components/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -493,82 +563,22 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
                           value: 'number',
                           id,
                         })
-                      )
-                    }
+                      )}
                     checked={variable.active_calculation === 'number'}
                     style={{ marginRight: 5 }}
                   />
                   # of Cases
                 </label>
               </form>
             )}
-
-            {variable.active_chart === 'survival' && (
```

Comment:
can these inputs just be commented out, so they're available when we get progress free plots? unless it's been moved somewhere else ... :)

Reply:
commented out inputs